Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/Controller.purs

```diff
@@ -165,12 +165,13 @@ eval Resend state = do
         _ <- pure $ toast (getString LIMIT_REACHED)
         _ <- pure $ toggleBtnLoader "" false
         continue newState{props{enterOTP = false}}
-      else exit $ ResendOTP newState
+      else do 
+        _ <- pure $ toast "OTP resent successfully"
```

Comment:
@kapilJhalani1 can we consume the string key here instead of using the string directly.